demo/gpu-test6.yaml

```diff
@@ -0,0 +1,48 @@
+# One pod: one initContainer + one container
+# Each asking for shared access to a single GPU
+
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: gpu-test6
+
+---
+apiVersion: resource.k8s.io/v1beta1
+kind: ResourceClaimTemplate
+metadata:
+  namespace: gpu-test6
+  name: single-gpu
+spec:
+  spec:
+    devices:
+      requests:
+      - name: gpu
+        deviceClassName: gpu.example.com
+
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  namespace: gpu-test6
+  name: pod0
+spec:
+  initContainers:
+  - name: init0
+    image: ubuntu:22.04
+    command: ["bash", "-c"]
+    args: ["export; trap 'exit 0' TERM; sleep 1 & wait"]
```

Comment:
Is there value in sleeping for 1 second over not sleeping at all? i.e. only run `export`?

Reply:
I suppose the value would be in better representing the work that an initContainer might do (probably not instantaneous?) But whatever best expresses and demonstrates a successful initContainer execution and then progression onto the primary container is fine, lemme know!

Reply:
I think a plain `export` is sufficient here. I don't think we need to simulate extra work in the init container to demonstrate the use case.